File: code-coaches/medium/safety-deposit-boxes/SafetyDepositBoxes.java
import java.util.Scanner;

public class SafetyDepositBoxes {

    public static int calculateTimeToGetItem(String[] items, String itemToFind) {
        int time = 5;
        for (String item : items) {
            if (!item.equals(itemToFind)) {
                time += 5;
            } else {
                break;
            }
        }

        return time;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] items = scanner.nextLine().split(",");
        String itemToFind = scanner.nextLine();

        System.out.println(calculateTimeToGetItem(items, itemToFind));

        scanner.close();
    }

}

File: code-coaches/medium/safety-deposit-boxes/safety_deposit_boxes.py
def calculate_time_to_get_item(items: list[str], item_to_find: str) -> int:
    time = 5
    for item in items:
        if item != item_to_find:
            time += 5
        else:
            break

    return time


def main():
    items = input().split(",")
    item_to_find = input()
    calculate_time_to_get_item(items, item_to_find)


if __name__ == "__main__":
    main()
